### PR TITLE
DIV 3981 desertion lived apart

### PR DESCRIPTION
**Pull request by mounikahmcts**
https://tools.hmcts.net/jira/browse/DIV-3981

Petitioner: Update 'Lived Apart Since' page to support  Desertion reason based on reasonForDivorceDesertionDate

### CODE REVIEW

**Comment by hosainnet** [comment]
This was approved 2 weeks ago, still needed? There are merge conflicts

File: app/steps/grounds-for-divorce/desertion/when-left/index.test.js [hunk 1]
```diff
@@ -147,7 +148,11 @@ describe(modulePath, () => {
           reasonForDivorceDesertionDate3YearsAgo.year()
       };
 
-      testRedirect(done, agent, underTest, context, s.steps.DesertionAgree);
+      if (config.features.release520) {
+        testRedirect(done, agent, underTest, context, s.steps.LivedApartSince);
+      } else {
+        testRedirect(done, agent, underTest, context, s.steps.DesertionDetails);
+      }
```
Comment (hosainnet) [inline app/steps/grounds-for-divorce/desertion/when-left/index.test.js]: Can we please follow https://github.com/hmcts/div-decree-nisi-frontend/pull/268 for testing config. In this case we'd probably split this into 2 tests

Comment (hosainnet) [reply app/steps/grounds-for-divorce/desertion/when-left/index.test.js]: For the sake of progress, I can fix is as part of DIV-4301 :)

**Comment by nsingamsetty** [comment]
Testing is completed and all looking good. Respective screenshots attached to ticket DIv-3981

**Pull request merged**